src/layers/layer.cpp

```diff
@@ -525,6 +529,44 @@ void Layer::setup_pointers()
                 get_child_names(),
                 ")");
   }
+
+  // Set whether this layer will run in-place
+
+  // Check for environment variable that disables this behavior
+  bool envvar_disable_inplace = false;
+  char* no_inplace = getenv("LBANN_NO_INPLACE");
```

Comment:
@tbennun This should use the options class.

Reply:
Updated, thanks!